branch-validation.yml

```diff
@@ -229,13 +227,13 @@
       when: master is not defined
 
     - name: floating ips are reachable
-      command: ping -c 5 {{ item.public_ip }}
+      command: ping -c 60 {{ item.public_ip }}
```

Comment:
Thanks for catching that 5 pings with 5 retries was consistently catching the server before it was actually back up, when it was responding to ping, but not ssh.  Decreasing the values and rebasing